pygeodiff/geodifflib.py

```diff
@@ -457,6 +456,16 @@ def read_changeset(self, changeset):
             raise GeoDiffLibError("Unable to open reader for: " + changeset)
         return ChangesetReader(self, reader_ptr)
 
+    def get_wkb_from_geometry(self, geometry):
+        func = self.lib.GEODIFF_createWkbFromGpkgHeader
+        func.argtypes = [ctypes.c_void_p, ctypes.c_char_p, ctypes.POINTER(ctypes.c_size_t)]
```

Comment:
how this even can work? GEODIFF_createWkbFromGpkgHeader has now 4 arrgs and int restype...

Reply:
but you want this line to reflect it too, no?